fms2_io/netcdf_io.F90

```diff
@@ -561,7 +564,7 @@ function netcdf_file_open(fileobj, path, mode, nc_format, pelist, is_restart, do
   logical :: success
 
   integer :: nc_format_param
-  integer :: err
+  integer :: err,IsNetcdf4=-999
```

Comment:
`IsNetcdf4` needs to have doxygen documentation.

Reply:
Done.